### DISCUSSION TITLE
分享一下总结的rclone挂载百度网盘，迅雷网盘的博客

### DISCUSSION THREAD

**Discussion by youtehub** [AlistGo/alist#1948] [Ideas]
耗时近一周调试出来的挂载，测试了一下下载速度在10M/s，感觉应该是最快速度了，麻烦的就是rclone调优，那几个脚本下载后自己结合人rclone官网的解释重新调整一下

博客地址：[https://blog.csdn.net/csdn_life18/article/details/127234864](https://blog.csdn.net/csdn_life18/article/details/127234864)

**Comment by im2058** [comment]
貌似又挂了。。。
rclone.exe mount BaiduYun:/ P: --cache-dir F:\RcloneCache --vfs-cache-mode writes --header "Referer:https://pan.baidu.com/"  --header "User-Agent:pan.baidu.com"
The service rclone has been started.
2022/10/11 21:41:39 ERROR : 新建文本文档.txt: Failed to copy: Method Not Allowed: 405 Method Not Allowed
2022/10/11 21:41:39 ERROR : 新建文本文档.txt: vfs cache: failed to upload try #1, will retry in 10s: vfs cache: failed to transfer file from cache to remote: Method Not Allowed: 405 Method Not Allowed

**Comment by Cyiming** [comment]
你好 你现在能用了么

**Comment by fordiy** [comment]
> 耗时近一周调试出来的挂载，测试了一下下载速度在10M/s，感觉应该是最快速度了，麻烦的就是rclone调优，那几个脚本下载后自己结合人rclone官网的解释重新调整一下
> 
> 博客地址：https://blog.csdn.net/csdn_life18/article/details/127234864

**Method Not Allowed: 405 Method Not Allowed**
同样问题，MP4和Zip文件都不能通过rclone上传百度云盘，请问有解决的吗，盼复！

**Comment by STEVENTAN100** [comment]
请问楼主百度网盘是有svip吗？如果是的话，我用alist的复制选项也能让速度在10-25MB/s这样，不需要rclone吧

**Reply by zerobeyondlogic** [reply]
怎样在命令行界面使用alist？

**Reply by lsrpanda** [reply]
是否可以直接用alist的webdav进行操作

**Comment by ititchuan** [comment]
405问题怎么解决呢？

**Reply by Iceber** [reply]
v3 的 bug，无解

**Reply by Iceber** [reply]
只能下载，不能上传，上传再安一个 v2，再挂到 v3 里估计可以

**Comment by daviddwlee84** [comment]
Just fixed the `Method Not Allowed: 405 Method Not Allowed` error

Here are key steps:

1. Web proxy: `disable` -> `enable`
2. WebDAV policy: `302 redirect` -> `native proxy`
3. Upload api: `https://api.alistgo.com` -> `https://d.pcs.baidu.com`
4. (optional: if not using admin) Role > Permissions: `Webdav read` & `Webdav manage`

---

tested on the latest beta version

```bash
docker exec -it alist bash
/opt/alist# ./alist version
Built At: 2025-09-04 14:18:24 +0000
Go Version: go1.25.1 linux/arm64
Author: Xhofe <i@nn.ci>
Commit ID: 930f9f6
Version: beta
WebVersion: dev
```
